go.graphics.swing/src/go/graphics/swing/AreaContainer.java

```diff
@@ -26,14 +26,14 @@
 
 import javax.swing.JPanel;
 
-import com.jogamp.opengl.GL2;
-import com.jogamp.opengl.GLAutoDrawable;
-import com.jogamp.opengl.GLCapabilities;
-import com.jogamp.opengl.GLEventListener;
-import com.jogamp.opengl.GLProfile;
-import com.jogamp.opengl.awt.GLCanvas;
-import com.jogamp.opengl.awt.GLJPanel;
-import com.jogamp.opengl.glu.GLU;
+import javax.media.opengl.GL2;
+import javax.media.opengl.GLAutoDrawable;
```

Comment:
build.gradle specifies 2.2.1 the latest version appears to be 2.3.1 as found here http://jogamp.org/deployment/jogamp-current/archive/Sources/ and the change went in February https://jogamp.org/bugzilla/show_bug.cgi?id=682#c19, so looks like just need to update the build.